packages/compass/src/main/auto-update-manager.ts

```diff
@@ -185,10 +186,18 @@ const checkForUpdates: StateEnterAction = async function checkForUpdates(
     if (fromState === AutoUpdateManagerState.UserPromptedManualCheck) {
       void dialog.showMessageBox({
         icon: COMPASS_ICON,
-        message: 'There are currently no updates available.',
+        message:
+          updateInfo.reason === 'outdated-operating-system'
+            ? `The version of your operating system is no longer supported. Expected at least v${updateInfo.expectedVersion}`
```

Comment:
I would maybe run this through product. Personally I'm not sure how common it is to prefix an OS version with `v` and it caught my eye 🙂 Maybe we can do a better wording here and also link to the docs (same link I mentioned above)

Reply:
A link here wouldn't be clickable, in which case a button on the message box would be better? 🤔

Reply:
Oh, right, it's a native box, skipped my mind. Hmmm, yeah, not sure, can try a button for this, but might be slightly weird too 🤔

Reply:
(Don't want to overcomplicate this too, so your call if it's too much extra effort)

Reply:
I added a button - it seems fine 👍